raiden/messages.py

```diff
@@ -1832,6 +1833,93 @@ def verify_request_monitoring(
         )
 
 
+class UnlockLightRequest(Message):
+    
+    def __init__(self, token_address: Address, channel_identifier: ChannelID, sender: Address, receiver: Address,
+                 merkle_tree_leaves: str, **kwargs):
+        super().__init__(**kwargs)
+        self.channel_identifier = channel_identifier
+        self.sender = sender
+        self.receiver = receiver
+        self.token_address = token_address
+        self.merkle_tree_leaves = merkle_tree_leaves
+
+    def __eq__(self, other):
+        return (
+            super().__eq__(other)
+            and isinstance(other, UnlockLightRequest)
+            and self.channel_identifier == other.channel_identifier
+            and self.sender == other.sender
+            and self.receiver == other.receiver
+            and self.token_address == other.token_address
+            and self.merkle_tree_leaves == other.merkle_tree_leaves
+        )
+
+    @classmethod
+    def unpack(cls, packed):
+        return cls(
+            token_address=packed.token_address,
+            channel_identifier=packed.channel_identifier,
+            receiver=packed.receiver,
+            sender=packed.sender,
+            merkle_tree_leaves=packed.merkle_tree_leaves
+        )
+
+    def pack(self, packed) -> None:
+        packed.channel_identifier = self.channel_identifier
+        packed.sender = self.sender
+        packed.receiver = self.receiver
+        packed.token_address = self.token_address
+        packed.merkle_tree_leaves = self.merkle_tree_leaves
+
+    def to_dict(self):
+        return {
+            "type": self.__class__.__name__,
+            "token_address": to_checksum_address(self.token_address),
```

Comment:
Here you should use to_normalized_address instead, same for the others

Reply:
Agree. But light client was using the checksum address as a key of the mapping holding the channels. I didn't wanted to touch it too much, so prefered to change it here.

Reply:
I get that, but this is for serializing and deserializing from a dictionary so you should be careful with that, i had problems with this before so i just let you know that sometimes when you pull down the node and the node starts again and try to deserialize this from database fails if you don't do it using the correct functions. If this works for you then it's ok but i just let you know that, that's all

Reply:
I know, that's why I needed to change it, (to serialize it as a checksum address). I don't remember it to fail, and changing this would need to change de LC.